### PR TITLE
Adding a package to help build and run BaaS from source

### PR DESCRIPTION
**Pull request by kraenhansen**
## What, How & Why?

Pushing the script I use to download and run BaaS from source.

## ☑️ ToDos
* [ ] 📝 Changelog entry
* [x] 📝 `Compatibility` label is updated or copied from previous entry
* [x] 🚦 Tests (manually - locally)
* [x] 🔀 Executed flexible sync tests locally if modifying flexible sync
* [x] 📦 Updated internal package version in consuming `package.json`s (if updating internal packages)
* [x] 📱 Check the React Native/other sample apps work if necessary
* [x] 📝 Public documentation PR created or is not necessary
* [x] 💥 `Breaking` label has been applied or is not necessary

### CODE REVIEW

**Review by kneth — APPROVED** [review]
I have tried it on my computer, and it seems to work fine.

File: integration-tests/baas-test-server/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Run BaaS from source
+
+Run the Lerna bootstrap from the root of this repository.
+
+```shell
+npx lerna bootstrap --scope @realm/baas-test-server
+```
+
+Create a `.env` file with your `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` secrets.
```
Comment (takameyer) [inline integration-tests/baas-test-server/README.md:9]: These should/could also be read from the local `env`

**Review by takameyer — COMMENTED** [review]
Mine failed
```
> Targets not specified. Assuming:
  node17-macos-arm64
> Error! No available node version satisfies 'node17'
error Command failed with exit code 2.
```

**Review by takameyer — APPROVED** [review]
I hit a few snags and left comments about that.  It would be nice to get the frontend running when you navigate to `localhost:9090`, but that's not a priority.  Just happy it works!

File: integration-tests/baas-test-server/README.md (hunk 1 above)
Comment (kraenhansen) [reply integration-tests/baas-test-server/README.md:9]: Sure, that's possible ... Although you _really_ shouldn't expose secrets in your global environment as it's (very!) easy to accidentally leak those to a rouge package / app running on your system.

File: integration-tests/baas-test-server/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Run BaaS from source
+
```
[the remark this answers is not in the document]
Comment (kraenhansen) [reply integration-tests/baas-test-server/README.md:2]: Great find - I'll submit a patch for that.

**Comment by kraenhansen** [comment]
@takameyer try pulling again, now it's building and exposing the UI too.

**Pull request merged**